akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -771,15 +771,27 @@ private[akka] class Shard(
         getOrCreateEntity(entityId)
         touchLastMessageTimestamp(entityId)
         ackTo.foreach(_ ! ShardRegion.StartEntityAck(entityId, shardId))
+      case WaitingForRestart =>
+        // fine, we just start it right away
+        log.debug("Request to start (already remembered) entity [{}]", entityId)
```

Comment:
is this the same as the case above? | them?

Reply:
It is, I wanted to provide different logging but I can do that anyways (and I didn't now), I'll fix.